tests/mir-opt/match/never_patterns.rs

```diff
@@ -0,0 +1,17 @@
+// skip-filecheck
+#![feature(never_patterns)]
+#![allow(incomplete_features)]
+
+enum Void {}
+
+// EMIT_MIR never_patterns.opt1.UninhabitedEnumBranching.before.mir
```

Comment:
Could this be a MIR building test, outputting the just-built mir ?

Reply:
Yes! How do I do that?

Reply:
Is it just `EMIT_MIR never_patterns.opt1.built.after.mir`?